pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
Tests are failing for this function, it looks like the name of the argument has changed in the function. I think this should fix it:
```suggestion
def divide(x, y):
```